Parsers/ASimWebSession/Tests/Fortinet_FortiGate_ASimWebSession_SchemaTest.csv

```diff
@@ -0,0 +1,103 @@
+Result
+"(0) Error: Missing recommended alias [HttpStatusCode] aliasing existing column [EventResultDetails]"
```

Comment:
can we fix this?

Reply:
we need to map:
HttpStatusCode = EventResultDetails

Reply:
The field has been added.